### ISSUE TITLE
There are no focus styles

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/extension-workshop#110]
Navigating with a keyboard is showing no focus styles. This is a big hit to accessibility.

To fix this we need to re-instate default focus styles, or alternatives which have likely been turned off.
